matlab/Glacier2/greeter/client.m

```diff
@@ -1,9 +1,6 @@
 % Copyright (c) ZeroC, Inc.
 
 function client(args)
-    % The Slice module VisitorCenter maps to a MATLAB namespace with the same name.
-    import VisitorCenter.*
```

Comment:
import namespace.* is discouraged in MATLAB. Here, it's clearer and simpler to not rely on import.